src/Executables/Game/World/Entities/PlayerEntity.cs

```diff
@@ -593,16 +608,15 @@ public override void SetPoint(EPoints point, uint value)
             switch (point)
             {
                 case EPoints.Level:
-                    Player.Level = (byte) value;
-                    foreach (var entity in NearbyEntities)
+                    var currentLevel = GetPoint(EPoints.Level);
+                    if (value > currentLevel)
                     {
-                        if (entity is IPlayerEntity other)
-                        {
-                            SendCharacterAdditional(other.Connection);
-                        }
+                        LevelUp((int) (value - currentLevel));
```

Comment:
Cant u just do `LevelUp(value - currentLevel)`? This covers increase and decrease without the if (?)

Reply:
Yeah I don't know what I was thinking 🫨